src/clear/model/json_deserialize.cr

```diff
@@ -60,16 +61,16 @@ macro columns_to_instance_vars
   # Create a new empty model and fill the columns from json
   #
   # Returns the new model
-  def self.from_json(string_or_io : String | IO)
-    Assigner.from_json(string_or_io).create
+  def self.from_json(string_or_io : String | IO, permit : Array(String) = [] of String, mass_assignment : Bool = true)
```

Comment:
```suggestion
  def self.from_json(string_or_io : String | IO, permit : Enumerable(String | Symbol) = [] of String | Symbol, mass_assignment : Bool = true)
```

Reply:
What is the difference between using Array / Enumerable. When should I use which?